gradle/libs.versions.toml

```diff
@@ -1,23 +1,23 @@
 [versions]
 
-kotlin = "1.8.10"
-gradle = "8.0.0"
+kotlin = "1.9.10"
+gradle = "8.2.0"
 gradle-nexus = "1.1.0"
-androidx-core = "1.9.0"
-androidx-test-rules = "1.4.0"
-camerax = "1.2.3"
-compose-bom = "2023.04.01"
-compose-compiler = "1.4.4"
-appcompat = "1.7.1"
-material = "1.6.1"
-lifecycle = "2.6.1"
-navigation = "2.5.2"
-accompanist = "0.25.1"
-coil = "2.2.2"
+androidx-core = "1.12.0"
+androidx-test-rules = "1.5.0"
+camerax = "1.3.1"
+compose-bom = "2023.10.01"
+compose-compiler = "1.5.6"
```

Comment:
Forgot about this one. The compiler needs to be 1.5.3 because of the kotlin compat

ref: https://developer.android.com/jetpack/androidx/releases/compose-kotlin

Reply:
Ah, missed that (I updated this first and then realized I needed to keep Kotlin at 1.9.10 due to dokka but didn't change the compiler version to accomodate)